tests/rustdoc-js-std/path-end-empty.js

```diff
@@ -0,0 +1,6 @@
+const EXPECTED = {
```

Comment:
Instead of this test, can you add a new one in `rustdoc-js` instead so we can check exactly what is returned? We have type with very long names so I don't think you need to update the sources for it, just pick a type and ensure its items are listed.

Reply:
it's not the length of the type that matters, it's the length of the associated item.

but yeah, it could probably use a more extreme testcase.

Reply:
Did you add the test?